app/src/main/java/com/elmeradrianv/shesafe/fragments/MapFragment.java

```diff
@@ -134,15 +184,85 @@ public void onLocationChanged(Location location) {
         if (location == null) {
             return;
         }
+        double longitudePFL = location.getLongitude() - currentLocation.getLongitude();
+        double latitudePFL = location.getLatitude() - currentLocation.getLatitude();
         currentLocation = location;
-        displayLocation();
+        LatLng speedVector = new LatLng(latitudePFL, longitudePFL);
+        LatLng actualLocation = new LatLng(
+                currentLocation.getLatitude(),
+                currentLocation.getLongitude());
```

Comment:
I am assuming these are used later?

Reply:
Yes, planning using in the next pull request